src/backend/efiling-api/jag-efiling-api.yaml

```diff
@@ -230,6 +230,25 @@ paths:
                 format: binary
         "404":
           description: Document not found
+  "/submission/{submissionId}/rushProcessing":
+    post:
+      summary: add rush processing to package
+      operationId: PostRushProcessing
+      tags:
+        - submission
+      parameters:
+        - $ref: "#/components/parameters/xTransactionId"
+        - $ref: "#/components/parameters/submissionId"
+      requestBody:
+        description: filing submit parameters
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/Rush"
+      responses:
+        "201":
+          description: Rush created
```

Comment:
failure case?

Reply:
Handled in code but re-added. We are pretty inconsistent with setting these.